web-client/src/presenter/actions/setValidationAlertErrorsAction.ts

```diff
@@ -86,6 +86,7 @@ export const setValidationAlertErrorsAction = ({
           }
         }),
     ),
+    scrollToErrorNotification: props.scrollToErrorNotification || false,
```

Comment:
This gives us a way explicitly telling the app when to scroll to the `ErrorNotification` banner. By default, we don't scroll. To change the default, we can call the above `setScrollToErrorNotificationAction`.